plugins/base/src/test/kotlin/content/annotations/SinceKotlinTest.kt

```diff
@@ -18,6 +28,216 @@ class SinceKotlinTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun versionsComparing() {
+        assert(Version("1.0").compareTo(Version("1.0")) == 0)
+        assert(Version("1.0.0").compareTo(Version("1")) == 0)
+        assert(Version("1.0") >= Version("1.0"))
+        assert(Version("1.1") > Version("1"))
+        assert(Version("1.0") < Version("2.0"))
+        assert(Version("1.0") < Version("2.2"))
+    }
+
+    @Test
+    fun `rendered SinceKotlin custom tag for typealias, extensions, functions, properties`() {
+        val configuration =   dokkaConfiguration {
+            extraOptions = listOf("-XXSinceKotlin")
```

Comment:
From the Kotlin docs:

> The "-X" and "-XX" flags are experimental and can be added and removed at any time.

Since `-XX` is part of the [hardcoded parsing](https://github.com/Kotlin/dokka/pull/2708/files#diff-89910d7fe7909fca5a75f09a3e3d7b446eaf94e342b48f79a4e9c59147a7ef87R279) in current code, this would mean that all flags are experimental and can be added and removed at any time, which isn't user friendly and might certainly differ from the behaviour that external plugin writers want for their own flags.

I think it's fine to add it to `-XXSinceKotlin`, since the feature is specific to stdlib and is not for average users, but making all flags experimental by default with no way of making it stable can lead to misunderstandings, so I think it should be dropped.

Reply:
It's better to change the name `FeatureOption` to `ExperimentalFeatureOption`. WDYT?